hooks/build

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Just a note, make sure this file is executable.

Reply:
Looks like a script doesn't need to be executable to run on docker hub https://hub.docker.com/repository/registry-1.docker.io/zappi/vault-init/builds/3f4fc68d-e624-4448-bdc8-abb6af3f6f28

Reply:
Noted.